requirements.txt

```diff
@@ -3,3 +3,4 @@ scipy
 ipython
 matplotlib
 cython
+h5py
```

Comment:
thoughts on adding this as a core dependency? (I guess it will likely be added through properties later on anyways??)